### PR TITLE
InlineHelp: Update icon colors in admin results

### PR DESCRIPTION
**Pull request by jancavan**
#### Changes proposed in this Pull Request

* This PR updates the icon colors in admin search results.

#### Testing instructions

1. Open the Help FAB and/or Customer Home
2. Type in a query. Example: `domain`
3. Make sure the icon colors look like the following:

## Help FAB:

**Normal state:**
Screen Shot 2020-07-08 at 11 39 49 AM

**Hover state:**
Screen Shot 2020-07-08 at 11 39 53 AM

**Active state:**
Screen Shot 2020-07-08 at 11 40 00 AM

## Customer Home

**Normal state:**
Screen Shot 2020-07-08 at 11 50 08 AM

**Hover state:**
Screen Shot 2020-07-08 at 11 50 11 AM

**Active state:**
Screen Shot 2020-07-08 at 11 50 17 AM

cc @retrofox

### CODE REVIEW

File: client/blocks/inline-help/style.scss [hunk 1]
```diff
@@ -383,14 +385,21 @@
 		}
 
 		.gridicon {
-			align-self: baseline;
+			align-self: center;
```
Comment (retrofox) [inline client/blocks/inline-help/style.scss:388]: Changing this puts the icon in the middle when the item is multiline.



If I'm not wrong, we want to align it at the top, do we

Comment (jancavan) [reply client/blocks/inline-help/style.scss]: I added it back in: https://github.com/Automattic/wp-calypso/pull/44004/commits/9bbd276842eaaff4d2a4b2e89a2a42a23cde0ad5, and made sure the icon doesn't shrink in 2-line results.

Comment (retrofox) [reply client/blocks/inline-help/style.scss]: image

**Comment by retrofox** [comment]
It seems to be misaligned in the customer homepage.

image

We should try to deal with these styles in the [inline-help component](https://github.com/Automattic/wp-calypso/blob/master/client/blocks/inline-help/style.scss) as the first option. However, if it isn't possible we can tweak how it looks in [its specific style file](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/customer-home/cards/features/help-search/style.scss).
For instance, we apply ellipsis in the `<HelpSearch />` component, which means that it never will get multi-lines.

**Pull request merged**